src/index.ts

```diff
@@ -108,28 +108,94 @@ interface BreakpointInfo {
 const ABORTED_ERROR_CODE = 409;  // google.rpc.Code.ABORTED
 
 /** @fires breakpointHit as soon as any breakpoints are hit */
-export class DebugProxy extends EventEmitter {
-  private wrapper: Wrapper;
+// We use "our" and "us" to refer this instance of `DebugProxy`.
+export interface DebugProxyInterface extends EventEmitter {
+  /**
+   * Updates the state of all pending breakpoints set by us.
```

Comment:
it would better to inline 'this DebugProxy instance' wherever you use 'us' or 'we'. It is more readable that way if a bit verbose.